cli/command_repository_set_parameters_test.go

```diff
@@ -182,6 +182,71 @@ func (s *formatSpecificTestSuite) TestRepositorySetParametersUpgrade(t *testing.
 	env.RunAndExpectSuccess(t, "index", "epoch", "list")
 }
 
+func (s *formatSpecificTestSuite) TestRepositorySetParametersDowngrade(t *testing.T) {
```

Comment:
This sounds like a negative test which would attempt to perform an invalid index downgrade and successfully hit a failure. Is that correct ? If so then shouldn't we be using `--index-version` operation here somewhere ?